lib/abi/token.json

```diff
@@ -14,7 +14,7 @@
       },
       {
         "indexed": false,
-        "name": "value",
+        "name": "amount",
```

Comment:
I changed this to match what I saw in the contract, is that accurate?

Reply:
Here it actually doesn't matter—the ABI library will just return the data with this same name, so it's your choice.

Different tokens will also name these differently (for the most out there, see [DAI](https://github.com/dapphub/ds-token/blob/master/src/token.sol#L24)'s), and their generated ABIs will be interchangable with other implementations.